### DISCUSSION TITLE
Default colors/fonts in tray

### DISCUSSION THREAD

**Discussion by alexlimbeck** [LibreHardwareMonitor/LibreHardwareMonitor#819] [General]
![immagine](https://user-images.githubusercontent.com/10768071/190366761-8a1eb2c8-542c-4a25-8267-7e6cb9065734.png)

As you can see in the photo, the default colors/fonts of LibreHardwareMonitor are wery bad....
In the photo you can see also the default colors/fonts of Open Hardware Monitor.

**Comment by meteorquake** [comment]
I didn't find that problem on installation. I presume it's a quirk relating to what fonts you have available? but it would need addressing. d

**Reply by alexlimbeck** [reply]
For testing purpose I just downloaded and installed the last version on my pc at work.
I selected two temperatures to be shown in the tray.
Take a look to the picture:

![Immagine](https://user-images.githubusercontent.com/10768071/212839807-65e48971-33d0-442f-92a6-8f27db55e2d5.png)

**Comment by meteorquake** [comment]
Ah that looks clearer what's happening, and now I see that in your original. I have reversed colours (black background) so the black edging doesn't appear and just the white of the digit shows. It may be the designer chose that style for the display to work on both normal and reverse colouring. I think it would be better to detect the background colouring in use and choose a default colour (with plain style) based on it rather than a single style that covers all. d
